composer.json

```diff
@@ -23,9 +30,9 @@
         "phpspec/phpspec": "^4.0",
         "phpstan/phpstan-shim": "^0.9.2",
         "phpunit/phpunit": "^6.5",
-        "se/selenium-server-standalone": "^2.52",
         "sylius-labs/coding-standard": "^2.0"
     },
+    "minimum-stability": "dev",
```

Comment:
Why do we need it?

Reply:
Since we provide Sylius in version `~1.3.0`, which basically means dev-master, `minimum-stability: dev` is needed